cypress/integration/form-validation/sectioned-form.spec.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="cypress" />
+
+
+describe('Form validation', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/in-a-sectioned-form');
+    });
+
+    it('should display page title', () => {
+        cy.get('[data-cy=pxb-toolbar]').should('contain', 'Sectioned Form');
+    });
+
+    it('should display required for fields on submit', () => {
+        cy.get('[data-cy=submit]').click()
+        cy.get('#name-field-helper-text')
+        .should('contain', 'Required')
+        cy.get('#address-field-helper-text')
+        .should('contain', 'Required')
+        cy.get('#city-field-helper-text')
+        .should('contain', 'Required')
+        cy.get('.MuiFormControl-fullWidth > .MuiFormHelperText-root')
+        .should('contain', 'Required')
+        cy.get('#zip-field-helper-text')
+        .should('contain', 'Required')
+        cy.get('#first-name-field-helper-text')
+        .should('contain', 'Required')
+        cy.get('#email-field-helper-text')
+        .should('contain', 'Required')
+    });
+
+    it('should display not required for fields after refresh', () => {
+        cy.get('[data-cy=submit]').click()
+        cy.get('#address-field-helper-text')
+        .should('contain', 'Required')
+        .invoke('css', 'color').should('equal', 'rgb(202, 60, 61)')
```

Comment:
Check for warn prop instead of color?

Reply:
removed css color check. Kept should can required as its passed from child prop